charts/rancher-monitoring/100.1.1+up19.0.3/charts/rancherServiceMetrics/templates/servicemonitor.yaml

```diff
@@ -0,0 +1,55 @@
+{{- $chartLabel := .Values.chartLabel }}
+{{- if eq $chartLabel "" }}
+{{- $chartLabel = (index (lookup "apps/v1" "Deployment" "cattle-system" "rancher").metadata.labels "chart") }}
+{{- end }}
+{{- if ne $chartLabel "" }}
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  labels:
+{{ include "kube-prometheus-stack.labels" . | indent 4 }}
+  name: rancher
```

Comment:
This name could be more descriptive

Reply:
Sure, although it is the service monitor for the rancher service.

Reply:
Is it for all Rancher things or only for metrics framework ?  `rancher-metrics-service-monitor` ? Maybe Arvind has suggestion.